language/Chinese_Simplified/strings.po

```diff
@@ -1,6 +1,6 @@
 # Chinese Simplified translations for MAME package
 # MAME 套件的简体中文翻译
-# Copyright (C) 1997-2025 MAMEdev and contributors
+# Copyright (C) 1997-2024 MAMEdev and contributors
```

Comment:
This looks incorrect. Copyright year should remain as 2025, instead of being changed back to 2024.